project-base/storefront/components/Layout/SimpleLayout/SimpleLayout.tsx

```diff
@@ -7,7 +7,7 @@ type SimpleLayoutProps = {
 
 export const SimpleLayout: FC<SimpleLayoutProps> = ({ heading, children, standardWidth }) => (
     <Webline>
-        <h1 className="text-center">{heading}</h1>
```

Comment:
`SimpleLayout` is used in three places. Please double-check if we want this to be global. If so, then it's fine.

Reply:
We currently don't have design for those other two places. Let's merge it and after someone will design users consent and new password page he can look at it